problemsets-mat223/linearalbegra.tex

```diff
@@ -5322,27 +5322,139 @@ \section*{Eigenvectors}
+			\begin{solution}
+				The key observation is that $A$ acts in a very simple way on 
+				vectors represented in the $\mathcal V$ basis, as we found in the
+				previous problem. In other words, for any vector $\vec v$, it is
+				simple to compute $A[\vec v]_{\mathcal V}$. We also know from the
+				previous part that $P^{-1}[\vec v]_{\mathcal E}=[\vec v]_{\mathcal V}$,
+				where $\mathcal E$ is the standard basis. 
+
+				So given any vector $\vec v$ represented by $[\vec v]_{\mathcal E}$
+				in the standard basis, we have
+				\[
+					[A\vec v]_{\mathcal E}=PAP^{-1}[\vec v]_{\mathcal E}
+				\]
+				since
+				\[
+					PAP^{-1}[\vec v]_{\mathcal E}
+					=PA[\vec v]_{\mathcal V}
+					=P[A\vec v]_{\mathcal V}
+					=[A\vec v]_{\mathcal E}.
+				\]
+
+
+			\end{solution}
 		\item Can you find a matrix $D$ so that
 			\[
 				PDP^{-1}=A?
 			\]
-		\item $[\vec x]_{\mathcal V}=\mat{1\\3\\4}$.  Compute $A^{100}\vec x$.
+			\begin{solution}
+				$D=\mat{2&0&0\\0&-1&0\\0&0&1}$ is such a matrix. 
+
+				We know, for example, that $A$ acts on $\vec v_1$ by multiplying
+				it by $2$. $P^{-1}\vec v_1=[\vec v_1]_{\mathcal V}=\mat{1\\0\\0}$,
+				so we should have that $D\mat{1\\0\\0}=\mat{2\\0\\0}=[2\vec v_1]_{\mathcal V}$. 
+				Then $P$ transforms this back into $[2\vec v_1]_{\mathcal E}$. 
+
+				By similar reasoning, the second and third columns of $D$ should be
+				$\mat{0\\-1\\0}$ and $\mat{0\\0\\1}$.  
+			\end{solution}
+		\item $[\vec x]_{\mathcal V}=\mat{1\\3\\4}$. Compute $A^{100}\vec x$.
+			\begin{solution}
+				$A^{100}\vec x=\mat{2^{100}\\3\\4}_{\mathcal V}$.
+
+				By the previous problem, we know how $A$ acts on vectors represented
+				in the $\mathcal V$ basis: it multiplies the first coordinate by
+				$2$, the second by $-1$, and leaves the third coordinate unchanged.
+				So we compute
+				\[
+					A^{100}\mat{1\\3\\4}
+					=A^{99}\mat{2\\-3\\4}
+					=A^{98}\mat{2^2\\3\\4}
+					=A^{97}\mat{2^3\\-3\\4}
+					=\dots
+				\]
+			\end{solution}
 	\end{parts}
 
 
 	\question
-	For an $n\times n$ matrix $T$, suppose its eigenvectors $\{\vec v_1,\ldots \vec v_n\}$
+	For an $n\times n$ matrix $T$, suppose its eigenvectors $\Set{\vec v_1,\ldots \vec v_n}$
 	form a basis for $\R^n$.  Let $\lambda_1,\ldots,\lambda_n$ be the corresponding
 	eigenvalues.
 	
 
 	\begin{parts}
-	\item Is $T$ diagonalizable (i.e., similar to a diagonal matrix)?  If so, explain how to obtain its diagonalized form.
-		\item What if one of the eigenvalues of $T$ is zero?  Is $T$ diagonalizable?
+	\item Is $T$ diagonalizable (i.e., similar to a diagonal matrix)? 
+		If so, explain how to obtain its diagonalized form.
+		\begin{solution}
+			Yes. 
+
+			Let $\mathcal V=\Set{\vec v_1,\ldots \vec v_n}$ be the basis consisting
+			of eigenvectors of $T$. Then $T$ acts on each of these by multiplying 
+			them by the appropriate eigenvalue: $T\vec v_i=\lambda_i\vec v_i$. 
+
+			So if $P$ is the transformation that takes vectors represented in the
```

Comment:
"Let `P` be the matrix that takes vectors..."

It's good to avoid starting paragraphs with if. It can lead so some awkward word gymnastics or long sentences...